src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -147,15 +151,18 @@ private PEModuleBuilder EmitModule
             get { return _factory.CompilationState.ModuleBuilderOpt; }
         }
 
-        public override BoundNode Visit(BoundNode node)
+        /// <summary>
+        /// Return the translated node, or null if no code is necessary in the translation.
+        /// </summary>
+        public override BoundNode? Visit(BoundNode? node)
```

Comment:
Should this also have `NotNullIfNotNull`?

Reply:
No.  A local declaration without an initializer is rewritten to null.

---
In reply to: [389174378](https://github.com/dotnet/roslyn/pull/42068#discussion_r389174378) [](ancestors = 389174378)